### DISCUSSION TITLE
V4.1 (AT32F4xx, USB-C)

### DISCUSSION THREAD

**Discussion by wrljet** [keirf/greaseweazle#340] [General]
The release notes for v1.15 mentions V4.1 (AT32F4xx, USB-C).

Is information about that hardware available somewhere?

Thanks,
Bill

**Comment by keirf** [comment]
It's an upcoming revision to the V4 design. Tweaked form factor, better mounting holes, USB-C connector, improved ESD protection. I'll be publishing details on the wiki very soon. The new design will be available for sale in a couple of months.

**Reply by shoka555** [reply]
Any news when the updated board will be available?
I'd be purchasing from your ebay store.
I've no immediate need for the device, so am happy to wait for the new release.
Cheers Harry

**Reply by keirf** [reply]
It will be available when I have sold my remaining stock of V4. Check back in October.

**Comment by wrljet** [comment]
Cool, thank you.
